### ISSUE TITLE
Research utilizing GitHub Pages for Galaxy Documentation & FAQ

### ISSUE THREAD

**Issue by guylepage3** [galaxypi/galaxy#56]
**Description**
Research utilizing GitHub Pages for Galaxy Documentation & FAQ that can be pulled in to galaxy.ooo and allow for quick and easy pull requests from the community.

**Research**
- [x] [Publish Your Project Documentation with GitHub Pages](https://blog.github.com/2016-08-22-publish-your-project-documentation-with-github-pages/)
- [x] [GitHub Pages Website](https://pages.github.com/)
- [x] Repository settings - https://github.com/galaxypi/galaxy/settings
<img width="500" alt="screenshot 2018-08-15 19 36 34" src="https://user-images.githubusercontent.com/1711854/44179144-68ac7680-a0c3-11e8-8296-3b160c56269a.png">
- [x] [Configuring a publishing source for GitHub Pages](https://help.github.com/articles/configuring-a-publishing-source-for-github-pages/)
- [x] [GitHub Pages Getting Started YouTube Tutorial](https://www.youtube.com/watch?v=RaKX4A5EiQo)
- [ ] Find out about CNAME records

**Comment by guylepage3** [comment]
<img width="600" alt="screenshot 2018-08-15 19 36 34" src="https://user-images.githubusercontent.com/1711854/44179144-68ac7680-a0c3-11e8-8296-3b160c56269a.png">

**Comment by faddat** [comment]
Guy, this is a great idea, however I must say I've had difficulties with GitHub pages in the past.  

Specifically, I am not sure how we will get it to build from our markdown files.  I think I have seen other projects use an external host to build pages from markdown, I'll have a look around and see if I can find an example to pass to you.

**Comment by guylepage3** [comment]
@faddat yes please sir.. This is not a priority at the moment anyway.. I am moving it to [Sprint 4](https://github.com/orgs/galaxypi/projects/4)
